src/main/java/io/kestra/plugin/fs/local/AbstractLocalTask.java

```diff
@@ -21,14 +19,22 @@
 @Getter
 @NoArgsConstructor
 public abstract class AbstractLocalTask extends Task {
+    static final String ALLOWED_PATHS = "allowedPaths";
 
-    @Schema(
-        title = "Allowed paths for local filesystem access",
-        description = "List of allowed paths for local filesystem access." +
-            " Tasks will only be able to operate on files within these paths."
-    )
-    @NotNull
-    protected Property<List<String>> allowedPaths;
+    protected List<String> getAllowedPaths(RunContext runContext) {
+        Optional<List<String>> allowedPathConfig = runContext.pluginConfiguration(ALLOWED_PATHS);
+
+        if (allowedPathConfig.isEmpty() || allowedPathConfig.get().isEmpty()) {
+            runContext.logger().warn("Missing 'allowed-paths' configuration. Task execution aborted to enforce secure file access.");
+
+            throw new SecurityException(
+                "'allowed-paths' configuration is required to enable access to the local filesystem. " +
+                    "Define at least one allowed path in the plugin configuration."
```

Comment:
I think there is a short URL that points to the plugin configuration doc, it would be worth adding it there.
@wrussell1999 do you remember it?

Reply:
Some months ago I asked to create a short-url pointing to plugin configuration so each time we use a plugin configuration in some plugin we can refer to this short-url so the user know how to set a plugin configuration.

This is a new configuration so there is no existing doc, but we do have a doc for plugin configuration in general.